apps/catalog/catalog-e2e/src/integration/dashboard/edit-movie.cy.ts

```diff
@@ -292,6 +297,11 @@ describe('Movie tunnel', () => {
     get('end-button').click();
     assertUrlIncludes(`c/o/dashboard/title/${movie.id}/main`);
     get('titles-header-title').should('contain', movie.title.international) + ' edited';
+    interceptEmail(`to:${e2eSupportEmail}`).then(mail => {
+      const subject = getSubject(mail);
+      expect(subject).to.include(`${movie.title.international} edited was submitted by ${org.name}`);
+      gmail.deleteEmail(mail.id);
```

Comment:
```suggestion
      return gmail.deleteEmail(mail.id);
```

I don't know if this change anything regarding promises on cypress ..

Reply:
same